src/components/control_form/index.js

```diff
@@ -58,13 +79,24 @@ export class ControlForm extends Component {
     this.setState({[name]: value})
   }
 
+  toggleProjectSave = () => {
+    let value;
+
+    this.props.currentProject.name
+      ? value = 'Update'
+      :value = 'Save';
+    
+      return value;
+  }
+
   render() {
+    // let project = this.loadProject();
```

Comment:
do we need this log?

Reply:
I don't remember if I removed this. Will on next PR if I didn't